stable/multus/templates/daemonset.yaml

```diff
@@ -0,0 +1,148 @@
+        volumeMounts:
+          - name: cnibin
+            mountPath: /host/opt/cni/bin
+            mountPropagation: Bidirectional
+      containers:
+      - name: kube-multus
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: [ "/usr/src/multus-cni/bin/multus-daemon" ]
+        resources:
+          {{- toYaml .Values.resources | nindent 10 }}
+        securityContext:
+          {{- toYaml .Values.securityContext | nindent 10 }}
+        terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+          - name: cni
+            mountPath: /host/etc/cni/net.d
+          - name: cnibin
+            mountPath: /opt/cni/bin
+          - name: host-run
+            mountPath: /host/run
+          - name: host-var-lib-cni-multus
+            mountPath: /var/lib/cni/multus
+          - name: host-var-lib-kubelet
+            mountPath: /var/lib/kubelet
+            mountPropagation: HostToContainer
+          - name: host-run-k8s-cni-cncf-io
+            mountPath: /run/k8s.cni.cncf.io
+          - name: host-run-netns
+            mountPath: /run/netns
+            mountPropagation: HostToContainer
+          - name: multus-daemon-config
+            mountPath: /etc/cni/net.d/multus.d
+            readOnly: true
+          - name: hostroot
+            mountPath: /hostroot
+            mountPropagation: HostToContainer
+          - name: multus-conf-dir
+            mountPath: /etc/cni/multus/net.d
+          # Dynamic primary CNI socket mount - configured by CAREN
+          - name: primary-cni-sock
+            mountPath: {{ .Values.daemonConfig.readinessIndicatorFile }}
+            readOnly: true
+        env:
+          - name: MULTUS_NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+      volumes:
+        - name: cni
+          hostPath:
+            path: /etc/cni/net.d
+        - name: cnibin
+          hostPath:
+            path: /opt/cni/bin
+        - name: hostroot
+          hostPath:
+            path: /
+        - name: multus-daemon-config
+          configMap:
+            name: {{ include "multus.fullname" . }}-config
+        - name: host-run
+          hostPath:
+            path: /run
+        - name: host-var-lib-cni-multus
+          hostPath:
+            path: /var/lib/cni/multus
+        - name: host-var-lib-kubelet
+          hostPath:
+            path: /var/lib/kubelet
+        - name: host-run-k8s-cni-cncf-io
+          hostPath:
+            path: /run/k8s.cni.cncf.io
+        - name: host-run-netns
+          hostPath:
+            path: /run/netns/
+        - name: multus-conf-dir
+          hostPath:
+            path: /etc/cni/multus/net.d
+        # Dynamic primary CNI socket volume - configured by CAREN
+        - name: primary-cni-sock
+          hostPath:
+            path: {{ .Values.daemonConfig.readinessIndicatorFile }}
```

Comment:
please add new line. 
Generally we have linters to catch this. this repo does not seems to have linters.